client/me/help/main.jsx

```diff
@@ -219,6 +224,51 @@ class Help extends React.PureComponent {
 		);
 	};
 
+	supportSessionCard = () => {
+		return (
```

Comment:
Suggestion: it's common to destructure any repetitive needed props at the top of a function (like you've done on L266): `const { translate } = this.props;`

Reply:
🎉